Client/src/Navigation/appbar.js

```diff
@@ -217,100 +149,171 @@ return RenderRoles;
           </Button>
           <Typography className={classes.spacer}>
           </Typography>
-         {role.toLowerCase()==='student' && 
-          <Button variant="text" href="/" className={classes.button}>Create Appointment</Button>}
-          <Button variant="text" href="/appointments" className={classes.button}>My Appointments</Button>
-          <Button variant="text" href="/FAQ" className={classes.button}>FAQ</Button>
+          {role.toLowerCase()==='student' && !isMobile &&
```

Comment:
I am not sure if this relates to this PR (if not, no action required) but since we are setting up the role in cookies throughout the app, ourselves and no input is expected from the user there, couldn't we just keep it in lower case by default? toLowerCase() is usually used when we can't be sure about the case difference, but here we could assure that.

Reply:
@ishchhabra Yeah for sure. We should check that it is always initialized the exact same way though first (we can just search the whole src directory for 'cookie'. Can you make this an issue?